_partials/_install-self-hosted-homebrew-based.mdx

```diff
@@ -0,0 +1,69 @@
+<Procedure>
+
+1.  Install Homebrew, if you don't already have it:
+
+    ```bash
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+    ```
+
+    For more information about Homebrew, including installation instructions,
+    see the [Homebrew documentation][homebrew].
+1.  At the command prompt, add the TimescaleDB Homebrew tap:
+
+    ```bash
+    brew tap timescale/tap
+    ```
+
+1.  Install TimescaleDB and psql:
+
+    ```bash
+    brew install timescaledb libpq
```

Comment:
I think the libpq package is just the C library -- not the psql client. right? Or am I mistaken?

Reply:
libpq has the psql client as well: https://www.timescale.com/blog/how-to-install-psql-on-mac-ubuntu-debian-windows.

Reply:
Oh, cool. TIL